src/main/java/com/google/swarm/tokenization/DLPS3ScannerPipeline.java

```diff
@@ -45,10 +46,13 @@ public static void main(String[] args) {
   public static PipelineResult run(S3ReaderOptions options) {
     Pipeline p = Pipeline.create(options);
 
-    PCollectionTuple nonInspectedContents =
-        p.apply(
-            "File Read Transforrm",
-            FileReaderTransform.newBuilder().setSubscriber(options.getSubscriber()).build());
+    PCollection<KV<String, String>> nonInspectedContents =
```

Comment:
Can we please return a tuple? Anywhere you have try/catch you should use multi output with error tag. I think there are quite a few places, you will need this change. Without multi output pipeline will fail to recover.